packages/zwave-js/src/lib/commandclass/ThermostatModeCC.ts

```diff
@@ -94,6 +94,11 @@ export class ThermostatModeCCAPI extends CCAPI {
 			throwWrongValueType(this.ccId, property, "number", typeof value);
 		}
 		await this.set(value);
+
+		if (this.isSinglecast()) {
+			// Verify the current value after a delay
+			this.schedulePoll({ property });
+		}
```

Comment:
Dumb question since I'm not that familiar w/ the code at all yet - should this also be setting it in the `valueDB`?

Reply:
Thermostat mode only has a single value. That gets stored in node.setValue()